### DISCUSSION TITLE
[UserStoriesToRefine] #10 As <user> I want a website that allows interested parties to connect with UBC via email

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#36] [General]
### As <user> I want a website that allows interested parties to connect with UBC via email

**Comment by christinagiancola** [comment]
**Level 2: As a cacao producer I can enter my email address, so that United by Chocolate can send me updates periodically.** 
Acceptance Criteria: On the homepage there is a text input that allows users to submit a valid email address to subscribe to newsletters. Users are notified that this subscription can be opted out of at any time.
